### ISSUE TITLE
Wrong businessDiff result on a specific case

### ISSUE THREAD

**Issue by bdinos** [kalmecak/moment-business-days#93]
```
const moment = require('moment-business-days')
const days = moment('2021-02-03T01:59:00-05:00').businessDiff(moment('2021-02-02T00:00:00-05:00'))
console.log(days)
```
Result is 2 instead of 1.

Is this a bug or have I misunderstood how `businessDiff()` works?

**Comment by mcdado** [comment]
It was fixed in 052a4a7c1413e7418e42b4d931857abc3903c2b1 but it hasn't been released yet. I'll ping @kalmecak

**Comment by bdinos** [comment]
Confirmed. It works on latest code.
